src/axes/axis.ts

```diff
@@ -129,6 +139,14 @@ export class Axis<D> extends Component {
     this._tickLabelContainer = this.content().append("g")
                                            .classed(Axis.TICK_LABEL_CLASS + "-container", true);
     this._baseline = this.content().append("line").classed("baseline", true);
+    this._annotationContainer = this.content().append("g")
+                                              .classed("annotation-container", true);
+    this._annotationContainer.append("g").classed("annotation-line-container", true);
+    this._annotationContainer.append("g").classed("annotation-circle-container", true);
+    this._annotationContainer.append("g").classed("annotation-rect-container", true);
```

Comment:
I didn't want to add more instance variables into the mix since I can easily access the values from accessing the `_annotationContainer`.  In this case, I do not necessarily need the `_annotationContainer` instance variable and probably access it via d3 code or just shelve these calls into function calls.

All in all, I don't think instance variables are needed here and if we don't want to see this code, we should probably shelve it into a function.

What do you think?